### ISSUE TITLE
[Android] Dispatch touch event from MarkerView to MapView

### ISSUE THREAD

**Issue by Hugoo13** [mapbox/mapbox-gl-native#5388]
Platform: Android
Mapbox SDK version: 4.1.0-beta.2

### Steps to trigger behavior
I tried to use MarkerView instead of Marker. When I want to scroll the map, if I touch a marker view, the scroll doesn't work. See here:
![ezgif com-video-to-gif 1](https://cloud.githubusercontent.com/assets/3592767/16150644/a61a7970-3498-11e6-99c7-d443a1bafd6b.gif)

### Expected behavior
When I begin to scroll the map with a finger on a MarkerView, the touch should be delegated to the map.

**Comment by tobrun** [comment]
@Hugoo13 thank you for reaching out. You are correct that this behaviour isn't correct. Will try patching this up and go for the same approach as with iOS. eg. handling click in core instead of the View object itself.

**Comment by tobrun** [comment]
I'm removing this from the `4.1.0` milestone, this change impacts the code a bit too much just before  release.

**Comment by tobrun** [comment]
Looking into this now, solution will be to let core determine pressed marker views instead of using the Android SDK onClick event.

**Comment by tobrun** [comment]
I'm working on a proposed solution in #5639

**Comment by tobrun** [comment]
This is on hold to merge until underlying logic is being sorted out

**Comment by tobrun** [comment]
Explicitly noting that issue depends on #6055 getting resolved first

**Comment by tobrun** [comment]
#6773 got merged to resolve #6055, going to rebase my PR in #6673 and see if it still works.

**Comment by tobrun** [comment]
#5639 was merged, closing
